lib/msf/util/document_generator/document_normalizer.rb

```diff
@@ -201,6 +201,16 @@ def normalize_references(refs)
             case ref.ctx_id
             when 'MSB'
               normalized << "* [#{ref.ctx_val}](#{ref.site})"
+            when 'BID'
+              normalized << "* [#{ref.ctx_id}-#{ref.ctx_val}](#{ref.site})"
+            when 'EDB'
+              normalized << "* [#{ref.ctx_id}-#{ref.ctx_val}](#{ref.site})"
+            when 'ZDI'
+              normalized << "* [#{ref.ctx_id}-#{ref.ctx_val}](#{ref.site})"
+            when 'WPVDB'
+              normalized << "* [#{ref.ctx_id}-#{ref.ctx_val}](#{ref.site})"
+            when 'PACKETSTORM'
+              normalized << "* [#{ref.ctx_id}-#{ref.ctx_val}](#{ref.site})"
```

Comment:
```suggestion
            when 'BID', 'EDB', 'ZDI', 'WPVDB', 'PACKETSTORM'
              normalized << "* [#{ref.ctx_id}-#{ref.ctx_val}](#{ref.site})"
```

Just coupling these together as they were duplicates.

Reply:
Just to confirm - are there any semantic changes with this PR? 👀 

It looks like this logic previously fell through to the `else` logic here, and formatted strings in the same way:

```ruby
            else
              normalized << "* [#{ref.ctx_id}-#{ref.ctx_val}](#{ref.site})"
            end
```

Reply:
You're right @adfoster-r7 ! I missed that part somehow. Thank you. I removed those changes.